packages/admin-next/dashboard/src/lib/client/payments.ts

```diff
@@ -1,5 +1,13 @@
-import { getRequest } from "./common"
+import { HttpTypes } from "@medusajs/types"
 import { PaymentProvidersListRes } from "../../types/api-responses"
+import { getRequest } from "./common"
+
+async function retrievePayment(id: string, query?: Record<string, any>) {
```

Comment:
suggestion: Should we migrate this to the `js-sdk` while we are at it?

Reply:
yes def! force of habit 😄